signalsamplemod.py

```diff
@@ -13,10 +27,10 @@
 MY_FIRST_INTERVAL = Interval.INTERVAL_1_MINUTE
 MY_SECOND_INTERVAL = Interval.INTERVAL_5_MINUTES
 TA_BUY_THRESHOLD = 18 # How many of the 26 indicators to indicate a buy
-PAIR_WITH = 'USDT'
-TICKERS = 'signalsample.txt'
-TIME_TO_WAIT = 4 # Minutes to wait between analysis
-FULL_LOG = False # List anylysis result to console
+PAIR_WITH = parsed_config['trading_options']['PAIR_WITH']

```

Comment:
Same thing here. Dont use same list, specifiy a new one for mods

Reply:
done, will be on next push - thank you!